### PR TITLE
Update generators and blueprints section

### PR DESCRIPTION
**Pull request by locks**
Fixes https://github.com/ember-cli/ember-cli/issues/7644.

### CODE REVIEW

File: _posts/2013-04-08-generators-and-blueprints.md [hunk 1]
```diff
@@ -85,38 +83,28 @@ installing
...
-With the `usePods` set to `true`.
 {% highlight javascript %}
 // .ember-cli
 {
     "usePods": true
 }
 {% endhighlight %}
 
-The following would occur when generating a route:
+To generate or destroy a blueprint in the classic while `usePods` is
```
Comment (kellyselden) [inline _posts/2013-04-08-generators-and-blueprints.md]: classic mode?

Comment (locks) [reply _posts/2013-04-08-generators-and-blueprints.md]: Thanks, I tweaked this section a bit more.

File: _posts/2013-04-08-generators-and-blueprints.md [hunk 2]
```diff
@@ -126,7 +111,12 @@ installing
   create app/taco/template.hbs
 installing
   create tests/unit/taco/route-test.js
+{% endhighlight %}
+
+To generate or destroy a blueprint in the classic structured while `usePods`
```
Comment (kellyselden) [inline _posts/2013-04-08-generators-and-blueprints.md]: structured -> structure

Comment (locks) [reply _posts/2013-04-08-generators-and-blueprints.md]: ✅

**Pull request merged**